keps/sig-api-machinery/2887-openapi-enum-types/README.md

```diff
@@ -351,6 +351,16 @@ const (
 
 Here, `StorageMedium` can have infinite number of possible values, which disqualify it as an enum type.
 
+The Go parser has a limitation on parsing type aliases. The parser cannot distinguish 
+between the original definition and its aliases. For example,
+```go
+type Foo string
+type FooAlias = Foo
+```
+would result in the parser to treat FooAlias and Foo as the same type. 
+As a result, `gengo` produce either `Foo` or `FooAlias` but not both.
+As a workaround, during beta graduation, the enum parser will be updated to accept any name of the type.
```

Comment:
I'm not super clear on what this means, what's the exact behavior that you think we can have? Maybe include the bug too where a lot of the discussion has been going on.

Reply:
Or can we try to be less hand-wavy on what we'll do here? I'm not sure I understand what you're proposing

Reply:
I agree. Externalizing this to an issue in `kubernetes/gengo`

Reply:
I'd still be happy if we could discuss here what we can expect as a solution.

Reply:
I just realized that this issue should affect other markers too. I will keep it short in the KEP and describe it in a separate issue